tasks/section2.yml

```diff
@@ -1,146 +1,278 @@
-- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled"
-  command: /bin/true
-  register: result
+- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled | chargen-dgram"
+  shell: "chkconfig --list | grep chargen-dgram |grep -q on"
+  register: chargen1_server_enabled_audit
+  failed_when: chargen1_server_enabled_audit.rc == 0
   always_run: yes
   changed_when: no
   ignore_errors: yes
   tags:
       - level1
-      - level2
       - audit
       - rule_2.1.1
 
-- name: "SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled"
-  command: /bin/true
+- name: "SCORED | 2.1.1 | AUDIT | Ensure chargen services are not enabled | chargen-stream"
+  shell: "chkconfig --list | grep chargen-stream |grep -q on"
+  register: chargen2_server_enabled_audit
+  failed_when: chargen2_server_enabled_audit.rc == 0
+  always_run: yes
+  changed_when: no
+  ignore_errors: yes
   tags:
       - level1
-      - level2
+      - audit
+      - rule_2.1.1
+
+- name: "SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled | chargen-dgram"
+  command: "chkconfig chargen-dgram off"
```

Comment:
The Service module not work here with:

TASK [RHEL7-CIS : SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled | chargen-dgram] ***
failed: [node1] (item=chargen-dgram) => {"failed": true, "item": "chargen-dgram", "msg": "systemd could not find the requested service \"'chargen-dgram'\": "}


[root@node1 vagrant]# chkconfig --list | grep chargen-stream

Note: This output shows SysV services only and does not include native
      systemd services. SysV configuration data might be overridden by native
      systemd configuration.

      If you want to list systemd services use 'systemctl list-unit-files'.
      To see services enabled on particular target use
      'systemctl list-dependencies [target]'.

	chargen-stream:	off

And here the ansible script:

- name: "SCORED | 2.1.1 | PATCH | Ensure chargen services are not enabled 
  service:
      name: "{{ item }}"
      state: stopped
      enabled: no
  with_items: chargen-dgram
  when: chargen1_server_enabled_audit.failed
  ignore_errors: yes
  tags:
      - level1
      - scored
      - ldap
      - services
      - patch
      - rule_2.1.1